db/migrations/0014_add_provider_rsr_scores.sql

```diff
@@ -0,0 +1,25 @@
+-- Migration number: 0014 	 2025-01-27T00:00:00.000Z
+CREATE TABLE IF NOT EXISTS provider_rsr_scores (
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS service_provider_stats (
```

Let's name this something like `service_provider_stats` so we can store more generalised data.